examples/kafka_reader/kafka_reader.go

```diff
@@ -22,7 +23,7 @@ func main() {
 
 	brokerList := strings.Split(*brokers, ",")
 
-	k := kafka.New(brokerList, *topic)
+	k := kafka.New(brokerList, *group, *topic)
```

Comment:
Let's keep kafka.New(brokerList, *topic) format, to denote the source, and add

k.Group = *group

This is more a client property.